### PR TITLE
Try to load trusted conv meta if we find no participants in the mention hud

### PR DESCRIPTION
**Pull request by buoyad**
Kind of already done. WIP because I may switch it to detect this case in `selectConversation` and request the trusted meta there. any thoughts @keybase/react-hackers?

### CODE REVIEW

**Comment by mmaxim** [comment]
How does this play out if it does miss? Is the HUD just sitting blank for like a second before everything pops in?

**Comment by buoyad** [comment]
@keybase/react-hackers  this is ready

File: shared/chat/conversation/input-area/user-mention-hud/mention-hud-container.js [hunk 1]
```diff
@@ -12,27 +14,57 @@ const mapStateToProps = (state: TypedState, {filter, conversationIDKey}) => {
...
 }
 
-export default compose(connect(mapStateToProps, () => ({}), mergeProps), setDisplayName('UserMentionHud'))(
-  MentionHud
-)
+export default compose(
+  connect(mapStateToProps, mapDispatchToProps, mergeProps),
+  setDisplayName('UserMentionHud'),
+  lifecycle({
+    componentDidMount() {
```
Comment (chrisnojima) [inline shared/chat/conversation/input-area/user-mention-hud/mention-hud-container.js]: we should avoid using lifecycle in new stuff. really doesn't flow check at all

**Comment by mmaxim** [comment]
Loading state seems pretty key, this could hit the network and be slow.

File: shared/chat/conversation/input-area/user-mention-hud/mention-hud-container.js (hunk 1 above)
Comment (buoyad) [reply shared/chat/conversation/input-area/user-mention-hud/mention-hud-container.js]: OK, added a wrapper for it. I think this component's typing is kinda messed up in general, I'll ticket taking a look at it.

**Pull request merged**